File: src/main/java/me/ctknight/uploadmanager/UploadNetworkException.java
/*
 * Copyright (c) 2016.  All rights reserved. Lai Jiewen <alanljw12345@gmail,com
 */

package me.ctknight.uploadmanager;

public class UploadNetworkException extends UploadException {

    public UploadNetworkException() {}

    public UploadNetworkException(String detailMessage) {
        super(detailMessage);
    }

    public UploadNetworkException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public UploadNetworkException(Throwable throwable) {
        super(throwable);
    }
}

File: lib/src/main/java/me/ctknight/uploadmanager/UploadNetworkException.kt
/*
 * Copyright (c) 2018. All rights reserved. Lai Jiewen <alanljw12345@gmail.com>
 */

package me.ctknight.uploadmanager

class UploadNetworkException : UploadException {

  constructor() {}

  constructor(detailMessage: String) : super(detailMessage) {}

  constructor(detailMessage: String, throwable: Throwable) : super(detailMessage, throwable) {}

  constructor(throwable: Throwable) : super(throwable) {}
}
